clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/UpsertGoogleLoadBalancerDescription.groovy

```diff
@@ -26,6 +29,8 @@ class UpsertGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescr
   String portRange
   String region
   String accountName
+  GoogleHttpLoadBalancer googleHttpLoadBalancer
```

Comment:
Maybe you can have some kind of shared base class between GoogleHttpLoadBalancer and a GoogleLoadBalancer (really 'network load balancer')? Something like both GoogleHttpLoadBalancer and GoogleNetworkLoadBalancer extending GoogleLoadBalancer, and GoogleLoadBalancer itself is abstract?
Just feels hinky to have both `description.portRange` and `description. googleHttpLoadBalancer.portRange` (same for the other similar fields).